example-data/turtle/sio-model/carePathway.ttl

```diff
@@ -12,4 +10,4 @@
   sio:SIO_000356 :process_ .
 
 :process_ a obo:NCIT_C142427, obo:NCIT_C159705, sio:SIO_000006 ;
-    prov:startedAtTime "2020-10-18T13:00:00"^^xsd:dateTime .
```

Comment:
You have different indentations within one document. We use four-space indentation in old versions.

Reply:
It is okay that two-space indentation will be used :D . But within one file, the indentations should be consistent.

Reply:
I fixed it. Lets see if we can add a checkers for it in gihub